src/config/createFilesConfig.json

```diff
@@ -0,0 +1,10 @@
+[
+	[ "package.json", "." ],
+	[ "typings.json", "." ],
+	[ "tsconfig.json", "." ],
+	[ "tslint.json", "." ],
```

Comment:
@matt-gadd questioned whether we want to distribute a `tslint.json` file as part of the app template? I question applied for the `.editorconfig`